src/redux/contacts/contacts.reducer.js

```diff
@@ -0,0 +1,27 @@
+import { SET_CONTACTS, SET_LOADING } from './contacts.types';
+
+const initialState = {
+  loading: true,
```

Comment:
It is better to have an initial value `loading`  = `false`, it is much more convenient